trainer/finetune.py

```diff
@@ -29,7 +29,7 @@
 flags.DEFINE_enum("size", "base", ["small", "base", "large", "3B", "11B"],
                   "model size")
 flags.DEFINE_string("name", "default", "name/description of model  version")
-flags.DEFINE_enum("mode", "all", ["train", "evaluate", "all", "export"],
+flags.DEFINE_enum("mode", "all", ["train", "evaluate", "all", "export", "probe_1", "probe_2"],
```

Comment:
same comment as above regarding the probe names.

the names "Probe1" and "Probe 2" are not very expressive.
I know that within the team we know what it means . But for the public audience visiting the repo it will not be as clear.